src/main/java/io/seqera/tower/cli/commands/LaunchCmd.java

```diff
@@ -209,6 +240,88 @@ private WorkflowStatus checkWorkflowStatus(String workflowId, Long workspaceId)
         }
     }
 
+    private List<Long> obtainLabelIDs(@Nullable Long workspaceId) throws ApiException {
+
+        if (labels == null || labels.isEmpty()) {
+            return Collections.emptyList();
+        }
+
+        if (!labelPermission(workspaceId)) {
```

Comment:
Note `launch` users cannot modify the given pipeline labels, but they must send those. According to this code, `launch` user won't be able to launch pipelines with labels. To summarize:
- Launch users must stick to the labels of the pipeline.
- Launch users cannot create new labels.